bpfman/src/command.rs

```diff
@@ -375,15 +365,12 @@ impl ProgramData {
         let id_rand = rng.gen::<u32>();
```

Comment:
Looking at this again I've realized we'll have issues here.
There's a `N in u32::max` chance  (where N is the number of loaded BPF progams in the system) that you'll generate an ID that's already in use, and then start overwriting keys in the DB 🤔 
It might be better to use `program_tmp_` as the prefix just in case.

Reply:
Good point.. 🤔  Let's use program_tmp_ for now